tests/PKSim.Tests/CLI/QualificationRunnerSpecs.cs

```diff
@@ -281,15 +281,15 @@ public void should_export_the_observed_data_mapping_relative_to_the_report_outpu
       {
          _mapping.ObservedDataMappings.Length.ShouldBeEqualTo(1);
          _mapping.ObservedDataMappings[0].Id.ShouldBeEqualTo(_observedData.Name);
-         _mapping.ObservedDataMappings[0].Path.ShouldBeEqualTo(FileHelper.CreateRelativePath(_expectedObservedDataCsvFullPath, _qualificationConfiguration.ReportConfigurationFile, true));
+         _mapping.ObservedDataMappings[0].Path.ShouldBeEqualTo($"OBS_DATA_FOLDER/{_observedData.Name}.csv");
       }
 
       [Observation]
       public void should_export_the_inputs_mapping_relative_to_the_report_output_folder()
       {
          _mapping.Inputs.Length.ShouldBeEqualTo(1);
          _mapping.Inputs[0].SectionId.ShouldBeEqualTo(_input.SectionId);
-         _mapping.Inputs[0].Path.ShouldBeEqualTo(FileHelper.CreateRelativePath(_expectedInputFullPath, _qualificationConfiguration.ReportConfigurationFile, true));
+         _mapping.Inputs[0].Path.ShouldBeEqualTo($"INPUTS/{PROJECT_NAME}/Simulation/{_input.Name}.md");
```

Comment:
sometimes explicit is better. In this case, the FileHelper.CreateRelativePath was not working so the text were OK!!